15-synthesis.Rmd

```diff
@@ -210,17 +210,18 @@ A next step in this case is to read-up on relevant articles in the area such as
 <!-- I think we want to be broader than that: being open source *enables* positive impact (RL) -->
 
 This is a technical book so it makes sense for the next steps, outlined in the previous section, to also be technical.
-However there are wider issues worth considering in this final section, which returns to our definition of geocomputation.
-One of elements of the term introduced in Chapter \@ref(intro) was that geographic methods should have a positive impact.
+However, there are wider issues worth considering in this final section, which returns to our definition of geocomputation.
+one of the elements of the term introduced in Chapter \@ref(intro) was that geographic methods should have a positive impact.
 <!-- This section is based on the premise that the key ingredients that make-up 'geocomputation', and maximize its ability to have a positive impact, depend on a wider . -->
 Of course, how to define and measure 'positive' is a subjective, philosophical question, beyond the scope of this book.
-Regardless of your worldview, consideration the impacts of geocomputational work is a useful exercise:
+Regardless of your worldview, consideration of the impacts of geocomputational work is a useful exercise:
 the potential for positive impacts can provide a powerful motivation for future learning and, conversely, new methods can open-up many possible fields of application.
-These considerations lead to the conclusion that geocomputation is part of a wider 'open source approach', engagement with which can lead to tangible benefits for the people and organizations doing geocomputation and the wider community.
+These considerations lead to the conclusion that geocomputation is part of a wider 'open source approach'.
+<!-- Engagement with this approach, and the community that supports it, can have tangible benefits. -->
 <!-- Consideration of real-world impacts leads to the conclusion that  -->
 
-As we saw in Section \@ref(what-is-geocomputation), other terms, including geographic information systems (GIS) and geographic data science (GDS), capture the range of possibilities opened-up by geospatial software.
-But geocomputation has advantages: it concisely captures the 'computational' way of working with geographic data advocated in this book --- implemented in code and therefore encouraging reproducibility --- and builds on desirable ingredients of its early definition [@openshaw_geocomputation_2000]:
+Section \@ref(what-is-geocomputation) presented other terms that mean roughly the same thing as geocomputation, including geographic information systems (GIS) and geographic data science (GDS).
```

Comment:
including GIScience and geographic data science.
GIS is just the tool but geocomputation, GIScience and GDS are more than just tools.

Reply:
Good point. Changed.